accounts/service.go

```diff
@@ -529,11 +561,32 @@ func (s *InterceptorService) TrackPayment(id AccountID, hash lntypes.Hash,
 		return nil
 	}
 
-	// Okay, we haven't tracked this payment before. So let's now associate
-	// the account with it.
-	account.Payments[hash] = &PaymentEntry{
-		Status:     lnrpc.Payment_UNKNOWN,
-		FullAmount: fullAmt,
+	if !ok {
```

Comment:
I'm wondering if we could ever end up here with an all-zero payment hash (e.g. because it wasn't set in one of the calls). Not sure if we should check for that at least before we associate the payment?

Reply:
Hmm I'm not completely sure what that I understood this correctly. I think this should be fixed with the fix for this other feedback:
https://github.com/lightninglabs/lightning-terminal/pull/642#discussion_r1334124267

But please double check if I misunderstood what you ment with this comment, and let me know if I need to fix something else :)